src/base-command.ts

```diff
@@ -52,18 +52,19 @@ export abstract class BaseCommand<T extends typeof Command> extends Command {
 
     try {
       await validateSnykToken(process.env.SNYK_TOKEN)
+      this.log(`✓ Valid Snyk Token.`)
     } catch (error) {
       this.error(`Invalid Snyk Token. ${error}`)
     }
     if (!process.env.TENANT_ID) {
       const accessibleTenants = await getAccessibleTenants()
       if (accessibleTenants.data.length === 0) {
         this.error(
-          'Not tenant accessible with your credentials. A Tenant is required for Universal Broker. Personal organizations are not compatible.',
+          'No tenant accessible with your credentials. A Tenant is required for Universal Broker. Personal organizations are not compatible.',
```

Comment:
nit: consistent capitalisation
```suggestion
          'No Tenant accessible with your credentials. A Tenant is required for Universal Broker. Personal organizations are not compatible.',
```